### ISSUE TITLE
Add skip to main content / search results link

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#873]
Also include "Found ### results" to indicate to screen readers number of results generated from search.

**Comment by sandbergja** [comment]
Note that blacklight upstream has a pretty sophisticated skip navigation link.  For example, if you tab through demo.projectblacklight.org/?search_field=all_fields&q=dog, you get:

1. skip to search
2. skip to main content
3. skip to first result

**Comment by kevinreiss** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/OrangelightBibdataRequests-571691cab409d8d821b873be/issues/pulibrary/orangelight/873?planning-poker) with Zenhub @maxkadel @rladdusaw @sandbergja
